src/util.js

```diff
@@ -0,0 +1,135 @@
+const fs = require("fs");
+
+const randomColor = () => {
+  const colors = ["green", "yellow", "blue", "red", "white", "pink"];
+  const randomColorsIndex = Math.floor(Math.random() * colors.length);
+
+  return colors[randomColorsIndex];
+};
+
+const title = (answers) => {
```

Comment:
Try to make your function names descriptive. A name like `title` would be assigned to a variable rather than a function. What does this function do? `generateTitle` perhaps, or `renderTitleHtml` ?